aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/BaseHandlerStd.java

```diff
@@ -37,9 +39,16 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
 
     protected static final String STACK_NAME = "rds";
     protected static final String RESOURCE_IDENTIFIER = "customdbengineversion";
-    protected static final String DEFAULT_ENGINE_NAME_PREFIX = "19";
-    protected static final String DEFAULT_ENGINE_NAME_SEPARATOR = ".";
     protected static final int RESOURCE_ID_MAX_LENGTH = 50;
+    protected static final String IS_ALREADY_BEING_DELETED_ERROR_FRAGMENT = "is already being deleted";
+    protected static final BiFunction<ResourceModel, ProxyClient<RdsClient>, ResourceModel> NOOP_CALL = (model, proxyClient) -> model;
+
+    protected static final Function<Exception, ErrorStatus> ignoreCEVBeingDeletedConditionalErrorStatus = exception -> {
+        if (isCEVBeingDeletedException(exception)) {
```

Comment:
Are we issuing Delete call twice?

Reply:
Mainly to handle out of band delete cases. This will help stack to overcome delete failed state.